src/components/Resize/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, JSX, useMemo, useEffect } from 'react';
+import CodeBlock from '@theme/CodeBlock';
+
+/**
+ * Resize Component - Simple demo page showing off SpringRoll's SafeScaleManager. Referenced in docs/Examples/resize.mdx
+ * @returns JSX.Element
+ */
+export default function ResizeExample(): JSX.Element {
+  const [width, setWidth] = useState(0);
+  const [height, setHeight] = useState(0);
+  const [ratio, setRatio] = useState(0);
+
+  const [manager, setManager] = useState(null);
+  // SafeScaleManager uses the window object which isn't available in Node
+  // so the build process requires it to be dynamically imported to avoid errors
+  useEffect(() => {
+
+    if (typeof window !== 'undefined' && window.speechSynthesis) {
```

Comment:
same note here about the speech synthesis

Reply:
Ah, I think the branch is just out of date. Going to update it.